src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const span = [...document.querySelectorAll('.population')];
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const spanArr = span.map((elem) => +elem.textContent.replace(/,/g, ''));
+const total = spanArr.reduce((acum, carrent) => acum + +carrent, 0);
```

Comment:
There is a typo in the variable name 'carrent'. It should be 'current' to reflect the correct spelling and improve code readability.